raiden_contracts/utils/proofs.py

```diff
@@ -231,12 +231,16 @@ def sign_one_to_n_iou(
         receiver,
         amount,
         expiration,
```

Comment:
Since we're changing the signature of this function anyway, could we use this opportunity to rename `expiration` to `expiration_block`? Then all arguments would be consistent with the JSON representation of the IOU. And it's also more descriptive.

Reply:
Good idea.  I'll do this in this PR.